src/main/scala/no/ndla/conceptapi/model/api/ConceptSearchParams.scala

```diff
@@ -11,20 +11,17 @@ import org.scalatra.swagger.annotations.{ApiModel, ApiModelProperty}
 
 import scala.annotation.meta.field
 
+// format: off
```

Comment:
Er denne berre oversett, eller vil du ha den der?

Reply:
La den på for jeg syns det var så vanskelig å lese denne pga de lange doc linjene.
Tenker den kan få ligge jeg :)